packages/dds/tree/src/test/shared-tree/fuzz/topLevel.fuzz.spec.ts

```diff
@@ -54,6 +54,7 @@ describe("Fuzz - Top-Level", () => {
 		abort: 1,
 		fieldSelection: { optional: 1, required: 1, sequence: 3, recurse: 3 },
 		schema: 1,
+		nodeConstraint: 1,
```

Comment:
Given that constraints are conditional on being in a transaction, we might want to set the weight a bit higher (maybe 3?).

Reply:
Updated weight to 3